src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <App />
```

Comment:
[OPTIONAL] Also, I'm sorry my partner did not mention it, but there is also a requirement to implement a stateless filter component that allows the user to filter data from the app somehow.

> Uses a Filter stateless component to filter data retrieved from the API.

I highly encourage you to do this, as it's a good exercise and part of the requirements.